CHANGELOG.rst

```diff
@@ -30,8 +30,6 @@ Improved prompts
 Improved terminal support
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
---------------
```

Comment:
I think we should keep this because it belongs to the changelog template - we have it between all releases.
It renders as horizontal line (`<hr>`) in HTML (in `build/user_doc/html/relnotes.html`, it took me a while to realize we renamed it).

As soon as we add some text to the "Improved terminal support" section above, the warning will go away.
Maybe there is a way to avoid or work around the warning?

Reply:
That's a good point. I did a test, and you are correct, adding ANYTHING to that last section silences that warning. 

Since the 3.3.0 section is a work-in-progress, perhaps add a "To-do" bullet-point, with the understanding that it's a placeholder for "real" info to be provided later? 

I just pushed another commit, a96bbb598, that does this. I think it looks good. What do you think? 

<img width="1098" alt="Screen Shot 2021-03-24 at 5 29 14 PM" src="https://user-images.githubusercontent.com/5694899/112391812-7805ee80-8cc6-11eb-8978-a9989b547f0e.png">